src/common/grpc.rs

```diff
@@ -1067,6 +1069,57 @@ impl GrpcError {
     }
 }
 
+#[derive(Debug)]
+pub struct ServerError {
```

Comment:
I don't quite get what you mean here? Are you saying this?
```
#[derive(Debug, Error)]
pub struct ServerError {
    grpc_error: GrpcError,
    #[source]
    cause: Option<anyhow::Error>,
}
```
The reason I didn't do that was because if a cause wasn't provided I wanted the source to be the `GrpcError`

Reply:
Never mind, I don't even know why I wanted that I'll change it